lib/repository/local_repository.dart

```diff
@@ -24,4 +24,9 @@ abstract class LocalRepository {
   Future<void> deleteStateHomePage();
 
   Future<String> getStateHomePage();
+
+  // Operations about StateSettingPanel 
+  Future<void> saveStateSettingPanel(String stateHomePage);
```

Comment:
The attribute here is more a stateSettingJson right?
```suggestion
  Future<void> saveStateSettingPanel(String stateSettingJson);
```